packages/sandbox/src/app/utils/isMaestro.ts

```diff
@@ -0,0 +1,3 @@
+export const isMaestro = () => {
+  return process.env.E2E === 'true';
+};
```

Comment:
https://maestro.mobile.dev/advanced/detect-maestro-in-your-app
This was not working?

Reply:
It's not JS function, and we don't have access to Socket 😅
So I did this workaround